drivers/gles3/storage/mesh_storage.cpp

```diff
@@ -285,6 +286,72 @@ void MeshStorage::mesh_add_surface(RID p_mesh, const RS::SurfaceData &p_surface)
 
 	ERR_FAIL_COND_MSG(!new_surface.index_count && !new_surface.vertex_count, "Meshes must contain a vertex array, an index array, or both");
 
+	if (GLES3::Config::get_singleton()->generate_wireframes && s->primitive == RS::PRIMITIVE_TRIANGLES) {
+		// Generate wireframes. This is mostly used by the editor
```

Comment:
```suggestion
		// Generate wireframes. This is mostly used by the editor.
```

Just a small comment style nitpick. We always end comments with a period.